code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -183,12 +184,13 @@
 	stored_mmi = new /obj/item/mmi/posibrain/ipc(src)
 	..()
 	spawn(1)
-		if(owner)
-			stored_mmi.name = "positronic brain ([owner.real_name])"
-			stored_mmi.brainmob.real_name = owner.real_name
-			stored_mmi.brainmob.name = stored_mmi.brainmob.real_name
-			stored_mmi.icon_state = "posibrain-occupied"
-			update_from_mmi()
-		else
-			stored_mmi.loc = get_turf(src)
-			qdel(src)
+		if(!QDELETED(src))
```

Comment:
This occurs in some situations where an IPC is quickly created then deleted---it wouldn't occur if it wasn't for the damn `spawn(1)`